android/src/app/organicmaps/bookmarks/BookmarkCategoriesAdapter.java

```diff
@@ -154,8 +154,23 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
   private void bindSize(@NonNull CategoryViewHolder categoryHolder,
                         @NonNull BookmarkCategory category)
   {
-    BookmarkCategory.CountAndPlurals template = category.getPluralsCountTemplate();
-    categoryHolder.setSize(template.getPlurals(), template.getCount());
+    if (category.size() == 0) {
```

Comment:
Please follow our code style guide.
```suggestion
    if (category.size() == 0)
    {
```